### ISSUE TITLE
how to retrain my own cocodatasets based on modal_final.pth pretrained???

### ISSUE THREAD

**Issue by tangze5258** [facebookresearch/maskrcnn-benchmark#987]
how to retrain my own cocodatasets based on modal_final.pth pretrained???
i add some new cocodatasets ,have modified  WEIGHT: "./model_final.pth(pretrained)", and then redo the "python train_net.py --config-file *.yaml",but it do not train, total training time =0,just skip the train process. .
how to deal with it ? thx..

**Comment by q417596917** [comment]
You need to delete or Modify this file with the name of

**Comment by q417596917** [comment]
hi,
you need to delete or modify this file with the name of last_checkpoint.

**Comment by tangze5258** [comment]
> hi,
> you need to delete or modify this file with the name of last_checkpoint.

@th512 thank you for your advice.Actually, i delete optimizer， scheduler and iteration dict and it did work when run train_net.py, but finally, has no new model_final.pth file generated,what's wrong ? thx....

**Comment by th512** [comment]
@tangze5258 
what are your running commands?  you just need to delete last_checkpoint.  "~/.torch/models/*.pkl",Is there a model in this path?
